tempodb/backend/s3/s3.go

```diff
@@ -660,6 +660,12 @@ func createCore(cfg *Config, hedge bool) (*minio.Core, error) {
 		opts.BucketLookup = minio.BucketLookupType(cfg.BucketLookupType)
 	}
 
+	clnt := &minio.Client{}
+
+	if cfg.UseDualStack {
+		clnt.SetS3EnableDualstack(false)
+	}
```

Comment:
This client is never returned. You need to apply the config to the client returned by this function `newCore()`.

Reply:
@mapno , I have modified the core and api logic. But `make vendor-check` both on local and on the workflow deletes the changes made. Can you help me here